### ISSUE TITLE
Running "interpreter --model i raises a LiteLLM:ERROR

### ISSUE THREAD

**Issue by ectormgl** [OpenInterpreter/open-interpreter#1512]
### Describe the bug

After running 
```interpreter --model i ```

without any previous configuration, the following error is raised (after some input is sent):

```shell
> wheres my chrime_analysis directory
16:16:08 - LiteLLM:ERROR: utils.py:1953 - Model not found or error in checking vision support. You passed model=i, custom_llm_provider=openai. Error: This model isn't mapped yet. model=i, custom_llm_provider=openai. Add it here - https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json.
``` 

### Reproduce

1.pip install open-interpreter
2. input at cmd:` interpreter --model i`
3. input up some question, like where is a random directory 

### Expected behavior

It should work sucesscully, without any errors and without the need of the configuration (as it would run with your hosted LLM)

### Screenshots

![image](https://github.com/user-attachments/assets/f4e8ba53-2395-4b0b-864c-49ce02b0389c)

### Open Interpreter version

0.4.3

### Python version

3.11.9

### Operating System name and version

Windows 11

**Comment by MikeBirdTech** [comment]
I agree that we should not have litellm check for pricing on the i model. To add clarity, OI still worked successfully and generated code for your request

**Comment by Maryam-Nasseri** [comment]
It gives the same error for me too, I exited with `ctrl + c` and re-typed `interpreter --model i`, this time even with the error, it proceeded to perform the task (just enter your question at the prompt). I believe it is an automatic system message that shows a non-existing error!

**Comment by interpreterwork** [comment]
Thank you for reporting this and for helping build the classic Python version of Open Interpreter. This issue concerns the earlier Python/LiteLLM/Jupyter product; this repository now tracks the Rust Open Interpreter built on Codex. We are closing the legacy report here so current product tracking starts clean. If you still want to pursue it, the community-maintained classic fork is https://github.com/endolith/open-interpreter. If the same problem affects the current Interpreter release, please open a new issue with the current template and include the version, provider, model, wire API, harness, platform, and reproduction.
